DashAI/back/models/parameters/models_schemas/SVC.json

```diff
@@ -1,21 +1,21 @@
 {
     "additionalProperties": false,
-    "error_msg": "Los parámetros de SVM deben ser alguno(s) de ['C', 'coef0', 'degree', 'gamma', 'kernel'  , 'max_iter', 'probability', 'shrinking', 'tol', 'verbose'].",
-    "description": "SVM es un algoritmo de aprendizaje supervisado, que clasifica información generando un hiperplano que separa los puntos de las clases de manera óptima.",
+    "error_msg": "SVM parameters must be any one(s) of ['C', 'coef0', 'degree', 'gamma', 'kernel' , 'max_iter', 'probability', 'shrinking', 'tol', 'verbose'].",
+    "description": "SVM is a supervised learning algorithm, which classifies information by generating a hyperplane that separates the points of the classes in an optimal way.",
     "properties": {
         "C": {
             "oneOf":[
                 {
-                    "error_msg": "El parámetro 'C' debe ser de tipo number positivo.",
-                    "description": "El parámetro 'C' es de regularización. Debe ser de tipo number positivo.",
+                    "error_msg": "The parameter 'C' must be of type positive number.",
+                    "description": "The parameter 'C' is a regularization parameter. It must be of type positive number.",
                     "type": "number",
                     "exclusiveMinimum": 0,
                     "default": 1
                 },
                 {
                     "items": {
-                        "error_msg": "El parámetro 'C' debe ser de tipo number positivo.",
-                        "description": "El parámetro 'C' es de regularización. Debe ser de tipo number positivo.",
+                        "error_msg": "The parameter 'C' must be of type positive number.",
+                        "description": "The parameter 'C' is a regularization parameter. It must be of type positive number.",
                         "type": "number",
                         "exclusiveMinimum": 0,
                         "default": 1
```

Comment:
What is the meaning of this nested property schema (and the similar others in this file)?

Reply:
I don't know, at least in the frontend I have always used the first part of the parameters with `oneOf[0]`.

I am not sure if it is used in the backend.